src/actions/AuthActions.js

```diff
@@ -11,11 +10,28 @@ import {
     GUEST_TOKEN_SET
 } from './types';
 
-import {FACEBOOK_AUTH} from '../../env';
+import {FACEBOOK_AUTH, FIREBASE_USER_PASSWORD} from '../../env';
```

Comment:
@amgoodman5 let me know what you think, but once users log in with facebook, we don't need a unique password for each one. I just made a very cryptic one and dropped it into `env.js` and will use that as the same PW for all users. Since we only need to identify them by email anyhow...

Reply:
🥇